api/IDBVersionChangeEvent.json

```
@@ -68,6 +68,56 @@
           "deprecated": false
         }
       },
+      "IDBVersionChangeEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBVersionChangeEvent/IDBVersionChangeEvent",
+          "spec_url": "https://w3c.github.io/IndexedDB/#dom-idbversionchangeevent-idbversionchangeevent",
+          "description": "<code>IDBVersionChangeEvent()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "25"
+            },
+            "firefox_android": {
+              "version_added": "25"
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
```

```suggestion
              "version_added": "10"
```

Based on mirroring, and also verifying https://mdn-bcd-collector.appspot.com/tests/api/IDBVersionChangeEvent/IDBVersionChangeEvent tests true in iOS 14.5